src/layouts/Header/UserButton/index.tsx

```diff
@@ -66,8 +67,17 @@ function UserButton({ auth }: Props) {
         onClick: handleLogout,
       },
     ];
+    const checkGender = () => {
+      if (!isEmpty(auth)) {
+        if (auth?.gender === 'male') {
+          return male;
+        } else {
+          return female;
+        }
+      }
+    };
```

Comment:
1. chổ này đổi thành toán tử 3 ngôi thử đi nè dùng if thế này k ổn
2. tạo thành cái biến thì hợp lý hơn